include/seqan3/core/bit_manipulation.hpp

```diff
@@ -96,4 +105,48 @@ constexpr uint8_t bit_scan_reverse(unsigned_t n)
 #endif
 }
 
+/*!\brief Convert the byte encoding of integer values to little-endian byte order.
+ * \ingroup core
+ * \tparam type The type of the value to convert; must model std::Integral.
+ * \param  in   The input value to convert.
+ * \returns the converted value in little-endian byte-order.
+ *
+ * \details
+ *
+ * This function swaps the bytes if the host system uses big endian and if the size of the integral types is not
+ * bigger than 8 byte. Otherwise, the function does nothing and returns the unchanged input value.
+ */
+template <std::Integral type>
+inline constexpr type enforce_little_endian(type const in) noexcept
```

Comment:
you don't need inline here

Reply:
Well, I know, but I wasn't sure, what we decided on? If we always put inline or not (even if the function is declared constexpr).

Reply:
no, no inline for constexpr functions (we do put it in front of global variables, but it makes a difference there).